packages/macro/src/index.ts

```diff
@@ -5,7 +5,27 @@ import MacroJS from "./macroJs"
 import MacroJSX from "./macroJsx"
 
 const config = getConfig({ configPath: process.env.LINGUI_CONFIG })
-const [i18nImportModule, i18nImportName = "i18n"] = config.runtimeConfigModule
+
+const getSymbolSource = (name: string) => {
+  if (Array.isArray(config.runtimeConfigModule)) {
+    if (name === "i18n") {
+      return config.runtimeConfigModule
+    } else {
+      return ["@lingui/react", name]
+    }
+  } else {
+    if (
+      Object.prototype.hasOwnProperty.call(config.runtimeConfigModule, name)
+    ) {
+      return config.runtimeConfigModule[name]
+    } else {
+      return ["@lingui/react", name]
+    }
+  }
+}
+
+const [i18nImportModule, i18nImportName = `i18n`] = getSymbolSource(`i18n`)
```

Comment:
can't this just be simple quotes? ''

Reply:
Oh yes it can, sorry - I have the habit of using backquotes everywhere, and it seems prettier doesn't change those. Will update.